### ISSUE TITLE
TypeScript Support

### ISSUE THREAD

**Issue by asp-entwickler** [jerrybendy/vue-touch-events#28]
main.ts
import Vue2TouchEvents from 'vue2-touch-events';
                                                  ^
Could not find a declaration file for module 'vue2-touch-events'.

**Comment by asp-entwickler** [comment]
Fast and dirty. May be useful for someone.

@jerrybendy You can close this issue. 
Anyway it would be nice to have full Typescript support.
Thanks.

remove .txt extension 
[TouchEvents.ts.txt](https://github.com/jerrybendy/vue-touch-events/files/2829431/TouchEvents.ts.txt)

**Comment by jerrybendy** [comment]
I'm not familier with TypeScript. Could you add a PR about all your changes?

**Comment by asp-entwickler** [comment]
My changes are not backward compatible.
I wrap you vueTouchEvents object according [Vue.js-Plugins with TypeScript](https://www.mistergoodcat.com/post/vuejs-plugins-with-typescript)

And it could not be used from vanilla JavaScript any more. 
Besides of that I used type <any> for fast solution, what is not good for production. So I got completely  separate file and maintain 2 files (.ts and .js) with the same functionality, in my opinion, is not good idea.

Right solution is to create types file (.d.ts)
I'll try to find a time and generate .d.ts ( tsc --declaration fileName.ts ) compatible with your .js but not manually. Or maybe someone will be faster than me.
